docs/rules/ultiduo.md

```diff
@@ -1,5 +1,5 @@
 # Ultiduo Ruleset
-This ruleset was made for Ultiduo 18/future Ultiduo competitions run by ozfortress.
+This ruleset was last updated for Ultiduo 18 - please consult with the ozfortress league admin team for any future Ultiduo competitions in case there has been any updates.
```

Comment:
The "last updated" note at the top isn't needed, our docs show when they were last updated. That section was mainly to note that this ruleset was added for UD18 and beyond.